packages/ssr-web/app/components/edit-screen/wrapper/index.hbs

```diff
@@ -0,0 +1,41 @@
+<div
+  class={{cn "edit-screen-wrapper" (if this.mq (concat "edit-screen-wrapper--" (if this.showPage "page" "modal")))}}
+  ...attributes
+>
+  {{!--
+    The modal has different styling and also different event listeners.
+    We don't want to support Escape to exit editing when presentation is a page.
+    --}}
+  {{#if (not this.mq)}}
```

Comment:
Is there a name for this that would make it more clear what’s happening? I gather this is shown until the media query has run…? 🤔

Reply:
Yeah, it's shown before the media query has run. I'll rename mq to something more descriptive.